spec/spec_helper.rb

```diff
@@ -7,6 +7,7 @@
 SimpleCov.start :rails do
   add_filter '/spec/'
   add_filter '/vendor/'
+  add_filter '/lib/'
```

Comment:
👍🏻

Reply:
I kinda disagree - if SlowQueryLogger is code we wrote ... why wouldn't we want tests?

Reply:
In this case we're implementing an interface we don't control, but I think you make a good point that it is worth testing `#call` is doing the right thing (there are ~4 branches in this class that'd be good to test).

Reply:
We tend to put things in `lib` that are not the sort of things that we test, e.g., rake tasks. In this case, I'm not sure I see the value in testing it -- really the only effective way to test it is to run it; everything else is "make believe" testing.

Reply:
I think a class we write could be considered different from a rake task, which is, more or less, a script we write with a DSL.   We don't write test code fro scripts, but we do write test code for classes.   That said, a comment as to why there are no tests might be considered.